diff --git a/configure.ac b/configure.ac
index edd184154..88a01ef3b 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1495,7 +1495,7 @@ fi
 
 if test x"$grub_mkfont_excuse" = x ; then
   # Check for freetype libraries.
-  AC_CHECK_TOOLS([FREETYPE], [freetype-config])
+  AC_CHECK_TOOLS([FREETYPE], ["pkg-config freetype2"])
   if test "x$FREETYPE" = x ; then
     grub_mkfont_excuse=["need freetype2 library"]
   fi
@@ -1558,7 +1558,7 @@ AC_SUBST([BUILD_WORDS_BIGENDIAN])
 
 if test x"$grub_build_mkfont_excuse" = x ; then
   # Check for freetype libraries.
-  AC_CHECK_PROGS([BUILD_FREETYPE], [freetype-config])
+  AC_CHECK_PROGS([BUILD_FREETYPE], ["pkg-config freetype2"])
   if test "x$BUILD_FREETYPE" = x ; then
     grub_build_mkfont_excuse=["need freetype2 library"]
   fi
